src/filehound.js

```diff
@@ -175,6 +192,15 @@ class FileHound extends EventEmitter {
         this.emit('end');
       });
   }
+
+  findSync() {
```

Comment:
might be worth extracting the async and sync logic to internal modules - findAsync.js and findSync.js